dashboard/urls.py

```diff
@@ -15,50 +15,61 @@
 """
 from django.conf.urls import url
 from django.contrib import admin
+
+from .common.util import *
+
+if getenv_bool('STUDENT_DASHBOARD_SAML', 'false'):
+    from django.contrib.auth.decorators import login_required
```

Comment:
need a bit of doc here -- not sure about the logic...
if "STUDENT_DASHBOARD_SAML" is true, login_required is imported, right?